sjsonnet/src/sjsonnet/Evaluator.scala

```diff
@@ -502,14 +502,20 @@ class Evaluator(resolver: CachedResolver,
   def visitMemberList(objPos: Position, e: ObjBody.MemberList, sup: Val.Obj)(implicit scope: ValScope): Val.Obj = {
     val asserts = e.asserts
     val fields = e.fields
-    var cachedSimpleScope: ValScope = null.asInstanceOf[ValScope]
+    var cachedSimpleScope: ValScope | Null = null
```

Comment:
question: Will this work in the same wya as before? 🤔

Reply:
Yes. It was failing all the tests due to some changes in how Scala 3 handles AnyVal.